client/src/partials/patient_invoice/registry/registry.js

```diff
@@ -133,15 +133,33 @@ function InvoiceRegistryController(Invoices, bhConstants, Notify, Session, util,
     load(vm.filters);
   }
 
+ //Call the opening of Modal
+  function openModal(invoice){
+    Invoices.openCreditNoteModal(invoice)
+    .then(function (success) {
+      if (success) {
+        Notify.success('FORM.INFO.TRANSACTION_REVER_SUCCESS');
+        return load();
+      }
+    });
+  }
+
   // Function for Credit Note cancel all Invoice
   function creditNote(invoice) {
-    Invoices.openCreditNoteModal(invoice)
-      .then(function (success) {
-        if (success) {
-          Notify.success('FORM.INFO.TRANSACTION_REVER_SUCCESS');
-          return load();
-        }
-      });
+    Cash.checkCashPayment(invoice.uuid).
```

Comment:
Remember to use `Notify` to handle HTTP errors!  What happens if the `Cash.checkCashPayment()` call fails?